### ISSUE TITLE
希望书中能讲讲深度学习的数据量可以小于参数个数的问题

### ISSUE THREAD

**Issue by A-Pai** [nndl/nndl.github.io#578]
希望书中能讲讲深度学习的数据量可以小于参数个数的问题
1、是这样吗？
2、为何可以这样

**Comment by dasepli** [comment]
这是一个学术界还在研究的问题，暂时应该还没有很好的理论解释。其实对于深度模型来说，“参数量”已经不是衡量一个模型容量(capacity)的合理方式了，现在学术界更想知道的是其“有效容量”(effective capacity)。暂时来说，深度学习的数据量和参数个数可能没有直接的数量上的大小关系，其之间的关系还与很多因素有关。

下面几篇论文可供参考：
[ICLR17] Understanding deep learning requires rethinking generalization (best paper)
[ICML17] A Closer Look at Memorization in Deep Networks
[STOC20] Does Learning Require Memorization? A Short Tale about a Long Tail
[NeurIPS20] What Neural Networks Memorize and Why: Discovering the Long Tail via Influence Estimation
[STOC21] When is Memorization of Irrelevant Training Data Necessary for High-Accuracy Learning?

自己也在关注这个问题，欢迎一块交流：）

**Comment by A-Pai** [comment]
谢谢解答

**Comment by xpqiu** [comment]
感谢反馈。旧版中的该问题会在《神经网络与深度学习（第二版）》中优化；若对应内容在第二版中已被重写或已不存在，将以第二版内容为准。这里先关闭该issue，第二版发布后请以新版为准。
